### PR TITLE
Ahmed

### PR DESCRIPTION
**Pull request by Muddoo**
NavBar

### CODE REVIEW

**Comment by makstheimba** [comment]
@Muddoo Hi!

Congratulations on starting the project!
I'll leave a few notes that could improve your team collaboration:
- Pull request title should reflect it's contents. In this particular case it could be named `Setting up the project and adding NavBar`. 
- Ideally we should complete one task per pull requests. It'll be easier for your teammates to understand what's happening and test your code
- Remember to ask your teammates to review your work :) This can really help the project find bugs. It'll also allow you and your teammates learn from each other

File: src/components/NavBar.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+function NavBar() {
+    const [isOpen, setIsOpen] = useState(false)
+    const [scroll, setScroll] = useState(false)
+
+    function handleClick() {
+        setIsOpen(!isOpen)
+    }
+
+    function handleScroll() {
+        window.scrollY >= 546 ? setScroll(true) : setScroll(false)
```
Comment (makstheimba) [inline src/components/NavBar.js:12]: Could you tell a little about what were you trying to achieve with `scroll` and `setScroll` variables? It's a little bit hard to understand what they are for.

My guess is that it is for screens with height less than 546. But as far as I know `window.scrollY` would show not the screen hight but scrollable height of the page - and I think with this we will never encounter case where `window.scrollY < 546`

Also it's a good practice to name all your constants so it's easy to understand what is happening if we were to modify this code in a few months :)

**Comment by makstheimba** [comment]
@Muddoo Thank you for setting up the project and for starting to work on it! 🎉

File: src/components/NavBar.js (hunk 1 above)
Comment (Muddoo) [reply src/components/NavBar.js:12]: scrollY is measuring the scroll value .. on page load scrollY = 0 and on scrolling value increases and at some point which is 546 the navbar style changes from dark to white

**Pull request closed without merging**